src/ResourceManager/Storage/ChangeLog.md

```diff
@@ -20,10 +20,18 @@
 ## Current Release
 
 ## Version 5.0.2-preview
-* Support create Storage Context with OAuth.
```

Comment:
Did you mean to remove this part of the changelog? I can't find it anywhere else so not sure where it was meant to be.

Reply:
Yes, this is not part ofr the storage management change